pkg/migrations/sql/idempotent/exemplar.sql

```diff
@@ -0,0 +1,122 @@
+-- get_exemplar_label_positions returns the position of label_keys as a one-to-one mapping with label_keys. It returns
+-- the positions of all label keys corresponding to that metric, so that it remains easier to add null values to those indexes
+-- whose labels are not present in the exemplar being inserted at the golang level.
+CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.get_or_create_exemplar_label_key_positions(metric_name_text TEXT, label_keys TEXT[])
```

Comment:
Any reason to add `_text` to variable names? Seems like an anti-pattern, its definitely not a convention used to write SQL in this project.

Reply:
Thats just to avoid long names like `get_or_create_exemplar_label_key_positions.metric_name`

Reply:
Even though the high level looks same, the actual code I don't think is common with `get_new_pos_for_key()`. Plus, its easier to understand keeping a diff func so I personally want this to be different.